src/Framework/NativeMethods.cs

```diff
@@ -624,9 +624,7 @@ internal static bool IsMaxPathLegacyWindows()
         }
     }
 
-    // CA1416 warns about code that can only run on Windows, but we verified we're running on Windows before this.
-    // This is the most reasonable way to resolve this part because other ways would require ifdef'ing on NET472.
-#pragma warning disable CA1416
```

Comment:
There should be a corresponding

```c#
#pragma warning restore CA1416
```

That should also be removed (for all of these).